### PR TITLE
Add gsi openssh 9.3 from fedora 39 current stable

### PR DESCRIPTION
**Pull request by fscheiner**
The changes are based on:

https://kojipkgs.fedoraproject.org//packages/gsi-openssh/9.3p1/4.fc39/src/gsi-openssh-9.3p1-4.fc39.src.rpm

### CODE REVIEW

**Review by msalle — APPROVED** [review]
I'll approve it, although I haven't gone (and cannot go) through all the changes, but the fact that the testing matrix looks good should be sufficient at this point.

File: gsi_openssh/source/addr.c [hunk 1]
```diff
@@ -297,6 +342,32 @@ addr_host_is_all0s(const struct xaddr *a, u_int masklen)
 	return addr_is_all0s(&tmp_result);
 }
 
+#if 0
```
[the remark this answers is not in the document]
Comment (johnny-mnemonic) [reply gsi_openssh/source/addr.c:345]: Yes, this seems to come from the sources in the Fedora package, as it's included in that commit. See:
https://github.com/gridcf/gct/pull/221/commits/08c8814447f9d34a73a243214463f0577617eecb#diff-ab2be537182b1782e07d7f30a01eb2a590292a3c50e5b28aee3547c16c0e26e8

[the remark this answers is not in the document]
Comment (fscheiner) [reply gsi_openssh/source/addr.c:345]: Yes, this seems to come from the sources in the Fedora package, as it's included in that commit. See:
https://github.com/gridcf/gct/pull/221/commits/08c8814447f9d34a73a243214463f0577617eecb#diff-ab2be537182b1782e07d7f30a01eb2a590292a3c50e5b28aee3547c16c0e26e8

[the remark this answers is not in the document]
Comment (msalle) [reply gsi_openssh/source/addr.c:345]: Actually directly from openssh: https://github.com/openssh/openssh-portable/blame/master/addr.c#L345

[the remark this answers is not in the document]
Comment (fscheiner) [reply gsi_openssh/source/addr.c:345]: Ok, didn't look that far.

**Pull request merged**